tools/importer/import.js

```diff
@@ -38,6 +38,7 @@ import createZPatternBlock from './rules/zPattern.js';
 import createMasonryBlock from './rules/masonry.js';
 import createMerchBlock from './rules/merchBlock.js';
 import createAsideBlocks from './rules/aside.js';
+import createCardsBlock from './rules/cards.js';
 // import tabsToBlocks from './rules/tabs.js';
```

Comment:
are these comments needed?

Reply:
we have written scriot for tabs but once prediction JSON is ready we will use this same script.